CONTRIBUTING.md

```diff
@@ -0,0 +1,103 @@
+# Contributing to Outstatic
+
+- Before jumping into a PR be sure to search [existing PRs](https://github.com/avitorio/outstatic/pulls) or [issues](https://github.com/avitorio/outstatic/issues) for an open or closed item that relates to your submission.
+- If you can't get something to work the way you expect or found a bug, [open a question](https://github.com/avitorio/outstatic/discussions/new?category=q-a) in our discussion forums.
```

Comment:
Proposed text:
> If you can't get something to work the way you expect, [open a question](https://github.com/avitorio/outstatic/discussions/new?category=q-a) in our discussion forums.

If someone accidentally opens an issue that should be a discussion, it's one click for us to move the item over to the discussion forum.